### PR TITLE
UserContent Base Class; linkTo UserContent/StaticSection/HiglassViewConfig

### PR DESCRIPTION
**Pull request by alexkb0009**

### CODE REVIEW

File: src/encoded/types/user_content.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            else:
+                return 'txt' # Default if no file extension.
+        return None
+
+
+
+
+@collection(
+    name='higlass-view-configs',
+    properties={
```
Comment (aschroed) [inline src/encoded/types/user_content.py]: Do you not also need/want a uniquekey here?

File: src/encoded/types/user_content.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        'title': 'HiGlass Displays',
+        'description': 'Dsiplays and view configurations for HiGlass',
```
Comment (aschroed) [inline src/encoded/types/user_content.py]: Typo - Dsiplays -> Displays

File: src/encoded/types/user_content.py (hunk 1 above)
Comment (alexkb0009) [reply src/encoded/types/user_content.py]: It appears to inherit user_content:name from UserContent w/o issue. Could explicitly add `unique_key="user_content:name"` to it though to add clarity. Seem if add `unique_key="static_section:name"` some errors start to be thrown on insert or create-mapping.

**Comment by alexkb0009** [comment]
Closing pull request, instead will put up for review + merge the combined PR of this and the permissions branch.

**Pull request closed without merging**